README.md

```diff
@@ -2,7 +2,7 @@
 
 ```yaml
   - name: Deploy
-    uses: deployer/deploy@v1
+    uses: deployphp/action@master
```

Comment:
From my understanding of github doc name be anything.

Reply:
This definitely failed as used in the README.md example, with an error: `Unable to resolve action `deployer/deploy@v1.0.4`, repository not found` changing to this fixes it.